Makefile

```diff
@@ -54,7 +54,9 @@ OPERATOR_DEV_CSV="0.0.1"
 # Export GO111MODULE=on to enable project to be built from within GOPATH/src
 export GO111MODULE=on
 
-build: gofmt golint govet
+build: gofmt golint govet binary
```

Comment:
I am used to all fashioned

all: for everything
dist: for binaries :)

But otherwise looks good to me.

Reply:
Since @cynepco3hahue doesn't mind, I'll keep the old target for backward compat and add `dist` as new target